### PR TITLE
setup mobile version skeleton

### PR DESCRIPTION
**Pull request by mandelatuks**
**In this project for the portfolio the following was done:**

-A hello section
-Add humbergar navbar
-Add HTML file
-Add CSS file
-Add images folders

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  
+    <main class="info">
+      <h2>Hey there. I’m <br> Mandela <br>
+ <span> I’m a software developer </span>
+</h2>
+  <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hesitate to contact me.
+  </p>
+
+</main>
```
Comment (gemmen29) [inline index.html:36]: Kindly try to match the `background image` with the Figma design and Please add the `arrow button`.

| Your design | Figma design |
| --- | --- |
| image | Template 4-main-mobile - Copy - Copy |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="mobile-container">
+    
+    <nav>
+    <div> 
+      <p class="viewmobile">Welcome
+        <i class="fa-solid fa-bars"></i>
+      </p>
```
Comment (gemmen29) [inline index.html]: Kindly change the `p` tag to be  `a` tag for semantic purposes.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  <a target="_blank" href="#">
+    <img src="images/Linkedinicon.svg" alt=" linked handles">
+  </a>
+
+  <a target="_blank" href="#">
+    <img src="images/github.svg" alt=" github handles">
+  </a>
+  
+  </div>
```
Comment (gemmen29) [inline index.html]: Kindly change the `div` tag to be `ul` tag and wrap each `a` tag with a `li` tag for semantic purposes.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+     
+    </nav>
+
+
+  </header>
+  
+    <main class="info">
+      <h2>Hey there. I’m <br> Mandela <br>
+ <span> I’m a software developer </span>
+</h2>
```
Comment (gemmen29) [inline index.html]: Kindly change this heading to be `h1` instead of `h2` since this is the main heading of the page.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  </a>
+
+  <a target="_blank" href="#">
+    <img src="images/github.svg" alt=" github handles">
+  </a>
+  
+  </div>
+
+</body>
+</html>
```
Comment (gemmen29) [inline index.html:62]: Kindly change the PR description to be more descriptive like mentioning what you did exactly in this PR like:
- Add navbar. 
- Add hero section.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @mandelatuks, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mandelatuks** [comment]
@gemmen29,
  I  corrected everything like in the requirement thanks for the guidance.

**Review by gemmen29 — APPROVED** [review]
Hi @mandelatuks, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**